packages/node-cli/src/core/sdksetup/SdkLicense.js

```diff
@@ -26,9 +27,14 @@ class SdkLicense {
 		if (process.env.npm_config_acceptsuitecloudsdklicense || process.env.npm_config_acceptSuiteCloudSDKLicense) {
 			return;
 		}
+		const tmpFilePath = os.tmpdir() + path.sep + LICENSE_FILENAME;
+		const fileExists = fs.lstatSync(tmpFilePath);
```

Comment:
From nodejs documentation seems that fs.lstatSync() throws if the entry is not found by default. Don't we need to change that behavior here or surround this call in try/catch block?

Reply:
You're right. I'll change it